.changelog/unreleased/improvements/1145-tendermint-boxed-header.md

```diff
@@ -0,0 +1,10 @@
+- [ibc-client-tendermint-types] Box header fields inside of Misbehaviour type so
+  that the type is smaller (i.e. trade size of the type for heap memory).  This
+  prevents stack overflows on systems with small stack (e.g. Solana).
+  ([\#1145](https://github.com/cosmos/ibc-rs/pull/1145))
+
+<!--
```

Comment:
Hey, @mina86. Just please remove these extra comments from the changelog, then we are good to go.
Thanks!

Reply:
D’oh.  I always forget about those.  Done.